internal/services/springcloud/spring_cloud_gateway_resource.go

```diff
@@ -182,30 +182,16 @@ func resourceSpringCloudGateway() *pluginsdk.Resource {
 							Type:     pluginsdk.TypeString,
 							Optional: true,
 							Default:  "1",
-							ValidateFunc: validation.StringInSlice([]string{
-								"500m",
-								"1",
-								"2",
-								"3",
-								"4",
-							}, false),
+							// There are additional options when the service team allow them
```

Comment:
```suggestion
							// NOTE: we're intentionally not validating this field since additional values are possible when enabled by the service team
```